.mergify.yml

```diff
@@ -34,7 +34,7 @@ pull_request_rules:
   - name: automatic merge
     actions:
       comment:
-        message: Thank you for contributing! Your pull request is now being automatically merged without squashing.
+        message: Thank you for contributing! Your pull request will be updated from master and then merged automatically without sqaushing (do not update manually).
```

Comment:
Typo

Reply:
Also, might be worthwhile to toss in this link as well, because ppl need to have "maintainers can push to fork" enabled (and it cannot be enabled in forks based in organizations).

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork

Reply:
fixed the typo. Not exactly understand how this link should be added. Perhaps you can just suggest a phrasing with this link inside?